.github/workflows/publish.yaml

```diff
@@ -47,8 +49,7 @@ jobs:
       - name: Update frontend dependencies
         run: make update/frontend-deps
```

Comment:
```suggestion
      - name: Update web dependencies
        run: make update/web-deps
```

Reply:
@knanao Addressed it. https://github.com/pipe-cd/pipecd/pull/3585/commits/d4611ce58b1eaf46bc22f312762d549a9e7fbcfd